spec/requests/admin/users/users_export_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe "/admin/users/export", type: :request do
+  let(:admin) do
+    create(:user, :super_admin,
+           name: "Admin1", username: "admin1", email: "admin1@gmail.com",
+           registered_at: "2020-05-06T13:09:47+0000")
+  end
+  let!(:user) do
+    create(:user, :org_member,
+           name: "John Doe", username: "john_doe", email: "john_doe@gmail.com",
+           registered_at: "2020-06-08T13:09:47+0000")
+  end
+
+  before do
+    sign_in(admin)
+    get "#{export_admin_users_path}.csv"
+  end
+
+  it "renders successfully" do
+    expect(response).to be_successful
+  end
+
+  it "renders the headers" do
+    expect(response.body).to include("Name,Username,Email address,Status,Joining date,Last activity,Organizations")
+  end
+
+  it "shows the correct number of total rows" do
+    puts response.body.lines
+    # This takes into account empty lines after each row
```

Comment:
I'm adding a task for myself to figure out how to remove the empty lines and update this test to reflect the actual number of rows + the header. I didn't spend too much time on this detail as I'd like to get the base out and then improve on it since this feature is behind a feature flag.